changelog.md

```diff
@@ -1,7 +1,53 @@
+# 2.0.0a1
+
+## Big Changes
+* We redesigned the scenario class completely. The scenario is implemented as a dataclass now and holds only environment variables (like limitations or save directory). Everything else was moved to the components directly.
+* We removed runtime optimization completely (no adaptive capping or imputing anymore).
+* We removed the command-line interface and restructured everything alongside. Since SMAC was building upon the command-line interface (especially in combination with the scenario), it was complicated to understand the behavior or find specific implementations. With the removal, we re-wrote everything in python and re-implemented the feature of using scripts as target functions.
+* Introducing trials: Each config/seed/budget/instance calculation is a trial.
+* The configuration chooser is integrated into the SMBO object now. Therefore, SMBO finally implements an ask-tell interface now.
+* Facades are redesigned so that they accept instantiated components directly. If a component is not passed, a default component is used, which is specified for each facade individually in the form of static methods. You can use those static methods directly to adapt a component to your choice.
+* A lot of API changes and renamings (e.g., RandomConfigurationChooser -> RandomDesign, Runhistory2EPM -> RunHistoryEncoder).
+* Ambiguous variables are renamed and unified across files.
+* Dependencies of modules are reduced drastically.
+* We incorporated Pynisher 1.0, which ensures limitations cross-platform.
+* We incorporated ConfigSpace 0.6, which simplified our examples.
+* Examples and documentation are completely reworked. Examples use the new ConfigSpace, and the documentation is adapted to version 2.0.
+* Transparent target function signatures: SMAC checks now explicitly if an argument is available (the required arguments are now specified in the intensifier). If there are more arguments that are not passed by SMAC, a warning is raised.
+* Instances, seeds, and budgets can be set to none now. However, mixing none and non-none will throw an exception.
```

Comment:
Does this mean that once we constraints budgets, seeds, and instances also need to be specified?